scripts/rpc.py

```diff
@@ -2059,6 +2059,19 @@ def bdev_lvol_shallow_copy_status(args):
     p.add_argument('src_lvol_name', help='source lvol name')
     p.set_defaults(func=bdev_lvol_shallow_copy_status)
 
+    def bdev_lvol_get_fragmap(args):
+        print_json(rpc.lvol.bdev_lvol_get_fragmap(args.client,
+                                                  lvs_name=args.lvs_name,
+                                                  lvol_name=args.lvol_name,
+                                                  offset=args.offset,
+                                                  size=args.size))
+    p = subparsers.add_parser('bdev_lvol_get_fragmap', help='Get lvol fragmentation map')
+    p.add_argument('lvs_name', help='lvs name')
+    p.add_argument('lvol_name', help='lvol name')
+    p.add_argument('offset', help='offset in bytes', type=int)
```

Comment:
maybe you can explain what offset and size stand for

Reply:
Updated the description and the document. Thank you.